src/components/Section/Section.css

```diff
@@ -25,39 +23,23 @@
   letter-spacing: 0em;
   text-align: left;
   margin: 0;
-  padding: 0;
+  padding: 0 0 70px 0;
 }
 
-.placeholder_vertical {
-  width: 575px;
-  height: 324px;
-  background-color: brown;
-
-  margin: 0;
-  padding: 0;
-}
-
-.placeholder_horizontal {
-  width: 100%;
-  height: 151px;
-  margin: 0;
-  padding: 0;
-  display: flex;
-  justify-content: space-between;
+@media screen and (min-width: 1281px) {
```

Comment:
Do you think this might be where the weird responsiveness is coming from?

Reply:
It is, and I fixed it now. The math in the calc statements wasn't working out, but I really don't know why. Anyway I tweaked the numbers and now it works.

Reply:
But... this is not really a great solution. I'm having a bit of trouble figuring out the best way to make the display still look good with more or less than four cards.  Played around with grid but it is tricky to make it work with variable numbers of columns. 

Using flex to center the cards is easy enough, but I'm not sure how to keep the title lined up with the leftmost card.
![image](https://user-images.githubusercontent.com/37159177/110417581-d777c380-8063-11eb-8aec-b3b5404fdd46.png)
I feel like I would need to introduce a container to hold both the title and the cardList. Thoughts?

Reply:
Could something like `grid-template-columns` help. I think it's fine to just wrap after 4 and keep it left aligned to same spot with less than 4 unless you prefer it centers stylistically. Both work! 

Maybe @TanyaKomar can weigh in to lend a hand here?

Reply:
Grid would work too if we are alright with having empty columns when there are not 4 items in a row

Reply:
And I just updated this one more time and I think it works fine, for 4 cards and for other numbers of cards.